code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,286 @@
+	M.adjustOxyLoss(-oxycalc, 0)
+	M.adjustToxLoss(oxycalc/2.5, 0)
+	if(prob(current_cycle) && M.losebreath)
+		M.losebreath--
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/convermol/overdose_process(mob/living/carbon/human/M)
+	metabolization_rate += 1
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/tirimol
+	name = "Tirimol"
+	description = "An oxygen deprivation medication that causes fatigue. Prolonged exposure causes the patient to fall asleep once the medicine metabolizes."
+	color = "#FF6464"
+	var/drowsycd = 0
+
+/datum/reagent/medicine/C2/tirimol/on_mob_life(mob/living/carbon/human/M)
+	M.adjustOxyLoss(-1)
+	M.adjustStaminaLoss(2)
+	if(drowsycd && (world.time > drowsycd))
+		M.drowsyness += 10
+		drowsycd = world.time + (45 SECONDS)
+	else if(!drowsycd)
+		drowsycd = world.time + (15 SECONDS)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/tirimol/on_mob_end_metabolize(mob/living/L)
+	if(current_cycle > 20)
+		L.Sleeping(200)
+	..()
+
+/******TOXIN******/
+/*Suffix: -iver*/
+
+#define FIZ_WEAKRATE	0.005
+
+/datum/reagent/medicine/C2/fiziver //fiz = phys ok?
+  name = "Fiziver"
+  description = "An antitoxin that temporarily weakens the user, making them susceptible to other forms of damage. Weakness and toxin healing scales with exposure."
+  overdose_threshold = 11 // Scales every damage so naturally quite abusable. Luckily the OD purges the chem which resets the damage mod *wink*
+  var/reset_mods = 0 //amount, not bool
+
+/datum/reagent/medicine/C2/fiziver/on_mob_life(mob/living/carbon/human/M)
+	var/maths = FIZ_WEAKRATE * current_cycle
+	reset_mods += maths
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod += maths
+	phis.burn_mod += maths
+	phis.oxy_mod += maths
+	phis.stamina_mod += maths
+	M.adjustToxLoss(round(reset_mods*-1000,0.01)) //Math is fun!
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/fiziver/on_mob_delete(mob/living/carbon/human/M)
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod = max(phis.brute_mod - reset_mods, 1)
+	phis.burn_mod = max(phis.burn_mod - reset_mods, 1)
+	phis.oxy_mod = max(phis.oxy_mod - reset_mods, 1)
+	return ..()
+
+/datum/reagent/medicine/C2/fiziver/overdose_process(mob/living/carbon/human/M)
+	var/maths = reset_mods*3
+	M.adjustBruteLoss(maths)
+	M.adjustFireLoss(maths)
+	M.adjustOxyLoss(maths)
+	..()
+	M.reagents.del_reagent(type)
+	return TRUE
+
+#undef FIZ_WEAKRATE
+
+/datum/reagent/medicine/C2/multiver //made up chem, it's a PALLETTE cleanser hehe
+	name = "Multiver"
+	description = "An antitoxin that scales with the more unique medicines in the body as well as purges chems (including itself). Causes lung damage."
+
+/datum/reagent/medicine/C2/multiver/on_mob_life(mob/living/carbon/human/M)
+	var/medibonus = 0 //it will always have itself which makes it REALLY start @ 1
+	for(var/r in M.reagents.reagent_list)
+		var/datum/reagent/the_reagent = r
+		if(istype(the_reagent, /datum/reagent/medicine))
+			medibonus += 1
+	M.adjustToxLoss(0.5 * medibonus)
```

Comment:
If I'm not mistaken, a positive adjustment means damage is dealt, the line should be 
`M.adjustToxLoss(-0.5 * medibonus)`

Reply:
you are correct

Reply:
Other thing I noticed, Aiuri is stated to cause minor eye damage, but at 1 point of eye damage per tick, 12 units are enough to blind you.

Reply:
Other thing I noticed, Aiuri is stated to cause minor eye damage, but at 1 point of eye damage per tick, 12 units are enough to blind you.

Edit: Eyes have 50 health points, and regenerate 0.05 per tick unless failing, which can be increased up to .2 per tick. 

So if you want to cure, let's say 20 units of burn damage, you need 40 ticks worth of aiuri. That would be 16 units. Since a medborg can only inject 5 units at a time, that would be 20 units, for 50 ticks. Since eyes regenerate passively at 0.05 units per tick, that's .95 eye damage per tick, for a total of 47.5 units of eye damage. Meaning after the aiuri has cleared from the system, the patient will be blind for (47.5 - 30) / 0.05 = 350 ticks. Since each tick is somewhere between 1s and 2s, I'll assume 1tick = 1.5s, so the patient will be blind for 525 seconds, which is 8 minutes and 45 seconds.

Not to mention that if you were to add another dose of aiuri in the above example, the patient would end up permanently blind. I'm all for making medical more complex, but this is insane.